src/todos.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
You can put your initial state here, instead of having an unused file.

Reply:
Ah I forgot to remove this file - I've taken it out now and put the initial state into a js file as mentioned earlier.